.github/actions/maven-build-and-tag/action.yml

```diff
@@ -184,7 +184,9 @@ runs:
 
     - name: Build and Test with Maven (and maybe Deploy)
       shell: bash
-      run: mvn ${{ steps.define_maven_command.outputs.command }} ${{ env.MAVEN_CLI_OPTS}} ${{ inputs.extra-maven-opts }}
+      run: |
+        mvn -T1C de.qaware.maven:go-offline-maven-plugin:resolve-dependencies ${{ env.MAVEN_CLI_OPTS}} ${{ inputs.extra-maven-opts }}
+        mvn ${{ steps.define_maven_command.outputs.command }} ${{ env.MAVEN_CLI_OPTS}} ${{ inputs.extra-maven-opts }}
```

Comment:
I see you added and then removed the --offline parameter on the second maven call: do we still need to download some dependencies on the regular build? (for plugins or tests maybe?)